test/unit/puppetlabs/services/master/master_service_test.clj

```diff
@@ -70,3 +69,50 @@
                   (test-path! "cert_inventory" "target/master-service-test/inventory.txt"))))))
         (finally
           (fs/delete-dir test-dir))))))
+
+; This ensures that a test failure will be logged for the version check
+; test if the master for some reason stops calling into the version check
+; library
+(def version-check-success
+  (atom false))
+
+(defn version-check-test-fn
+  [request-values update-server-url]
+  (is (= {:group-id "puppets" :artifact-id "yoda"} (:product-name request-values)))
+  (is (= "http://notarealurl/" update-server-url))
+  (swap! version-check-success not))
```

Comment:
This is purely a style comment, so feel free to ignore.  I'd be inclined to have this just set the atom with the contents of the arguments passed into the function - maybe as a map with a key/val pair for each argument - and move these `is` assertions into the body of the deftest.  Where possible, I like to have assertions inside the body of the `deftest` so that what it is testing is more prominent.